cloudigrade/config/settings/prod.py

```diff
@@ -86,7 +86,11 @@ def django_traces_sampler(sampling_context):
         send_default_pii=True,
     )
 
-# Specifically in production, this default should be False.
+# Specifically in production, these defaults should be False.
```

Comment:
Shouldn't this be the other way around? We _do_ want this enabled in prod, right?

I know this gets set/overridden later in the `clowdapp.yaml`, but maybe it's finally time for this (and the `SOURCES_ENABLE_DATA_MANAGEMENT_FROM_KAFKA` setting below it) to have a more reasonable default.

Thoughts?

Reply:
I'll +1 @infinitewarp, I think this functionality should be enabled by default in all environments, it seems fundamental to cloudigrade not retaining stale data.

Reply:
makes sense  👍   I'll update both to default to True.